frontend/src/metabase/collections/components/CollectionSidebar/Bookmarks/getIcon.ts

```diff
@@ -0,0 +1,18 @@
+import { BookmarkableEntities } from "metabase-types/api";
+
+export function getIcon(
+  display: string | undefined,
+  type: BookmarkableEntities,
+) {
+  if (display) {
+    return display;
+  }
+
+  const icons = {
+    card: "grid",
+    collection: "folder",
+    dashboard: "dashboard",
+  };
+
+  return icons[type];
```

Comment:
Can we also use the "model" icon for models? FYI, `if (display) {}` check will pass for models since they're essentially cards, but they always have `display` set to `"table"`.

Reply:
Looks like the backend doesn't distinguish cards and models 🤔

Reply:
@kulyk Yes, we need a quick round on models, I'll check it out right after my current PR in the works.